<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sinif_islemleri</class>
 <widget class="QMainWindow" name="sinif_islemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sınıf işlemleri</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(234, 215, 187);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(17, 57, 70);</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>89</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="ekleme_btn_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(17, 57, 70);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Ekleme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="silme_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(17, 57, 70);
background-color: rgb(255, 0, 4);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Silme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="degistirme_btn">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Değiştirme</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Sınıf işlemleri</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
